ui/app/routes/observability/inferences/$inference_id/route.tsx

```diff
@@ -398,6 +413,23 @@ export default function InferencePage({ loaderData }: Route.ComponentProps) {
     }
   }, [humanFeedbackFetcher.data, humanFeedbackFetcher.state, navigate]);
 
+  const config = useConfig();
+
+  const isDefault = inference.function_name === "tensorzero::default";
+
+  const modelsSet = new Set<string>([
+    // models successfully used with default function
+    ...model_inferences.map(({ model_name }) => model_name),
+    // all configured models in config
+    ...Object.keys(config.models),
+    // TODO(bret): list of popular/common model choices
+    // see https://github.com/tensorzero/tensorzero/issues/1396#issuecomment-3286424944
+  ]);
+  const models = [...modelsSet].sort();
```

Comment:
Using a Set here to ensure there aren't any duplicates. For now, I thought alphabetical search would be sufficient.

Should the number of options grow to double digits, leveraging a dropdown that has a search field would be reasonable to add here